src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/programsBuilder.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import type StorageController from 'capture-core-utils/storage/StorageController';
+import { typeof StorageController } from 'capture-core-utils/storage';
```

Comment:
Basically here the StorageController is not a flow file. After the flow update in cases like this we need to have the `typeof` next to the import